install/addons/syndesis-server-rest-api-dashboard.yml

```diff
@@ -0,0 +1,3489 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: syndesis-server-rest-api
+  spec:
+    name: syndesis-meta.json
```

Comment:
Need to update the file name to reflect the dashboard we're configuring. So `syndesis-server-rest-api.json`.

Reply:
FIxed.